src/utils/arcgis-auth.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  getAuthenticatedUser,
+  arcGisRequestLogin,
+  arcGisRequestLogout,
+} from "./arcgis-auth";
+
+jest.mock("@esri/arcgis-rest-request");
+
+const ARCGIS_REST_USER_INFO = "__ARCGIS_REST_USER_INFO__";
+const mockEmailExpected = "usermail@gmail.com";
+
+let OLD_ENV = {};
+
+beforeAll(() => {
+  OLD_ENV = process.env;
+});
+
+describe("getAuthenticatedUser", () => {
```

Comment:
The test suite is testing "getAuthenticatedUser", "arcGisRequestLogin" and "arcGisRequestLogout", but the describe is about only `getAuthenticatedUser`